doc/U-19_Initialize_game.md

```diff
@@ -0,0 +1,21 @@
+# U-19: Initialize game
+**Actor:** Admin
+
+**Preconditions:** -
+
+**Postconditions:**
+
+ - Team status for every cypher is reset (first is in status pending)
+ - Every team has no taken hint
+ - Teams have 0 points
+
+## Main Scenario
+
+ 1. **Admin:** decides to start the game
+ 2. **System:** asks for confirmation
+ 3. **Team:** confirm the action
```

Comment:
Idea: enter the admin's password to confirm the action (because this action is quite critical)

Reply:
Good idea.